ereg-mapper-core/src/main/java/no/nav/registre/ereg/domain/Flatfil.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.registre.ereg.domain;
+
+import java.util.Optional;
+
+abstract class Flatfil {
+
+    String trimAndEmptyToNull(String value) {
+        if (value.trim().length() == 0) return null;
```

Comment:
Skriv med nye linjer for if:
if (value.trim().length() == 0) {
   return null;
}

synes det er lettere å lese